sdk/runanywhere-react-native/packages/core/android/src/main/java/com/margelo/nitro/runanywhere/PlatformAdapterBridge.kt

```
@@ -0,0 +1,375 @@
+                urlConnection.setRequestProperty("Prefer", "resolution=merge-duplicates")
+            }
+
+            // Write body
+            urlConnection.outputStream.use { os ->
+                os.write(jsonBody.toByteArray(Charsets.UTF_8))
+            }
+
+            val statusCode = urlConnection.responseCode
+            val responseBody = try {
+                urlConnection.inputStream.bufferedReader().use { it.readText() }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)